### PR TITLE
[TE] Anomaly ID Presentation Component

### PR DESCRIPTION
**Pull request by newsummit**
Adding the anomaly title and id header block (this may be changing soon) along with styles and integration test for display of title, ID, percent change, etc.

### CODE REVIEW

**Comment by justYves** [comment]
@apucher Good question! 

TLDR: Yes
---

```
Yarn maintains consistency across machines in two key ways:

Yarn uses a deterministic algorithm that builds up the entire dependency tree before placing files where they need to be.
Important info from the install process is stored in the yarn.lock lockfile so that it can be shared between every system installing the dependencies.
```
source: https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

File: thirdeye/thirdeye-frontend/app/styles/components/anomaly-id.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.anomaly-id {
+  list-style: none;
+  font-size: 15px;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/components/anomaly-id.scss]: Todo: we should save all these magic numbers in a `variable.scss` files.

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+});
+
+const idSelector = '.anomaly-id__id';
+const nameSelector = '.anomaly-id__name';
+const currentValueSelector = '.anomaly-id__total';
+const changeRateSelector = '.anomaly-id__percent';
+const anomalyId = 20475;
+const anomalyCurrentValue = 10;
+const anomalyBaselineValue = 2;
+const anomaly = Ember.Object.create({
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: just curious does this need to be an ember object?

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+const calculateChangeRate = (current, baseline) => {
+  return (baseline === 0) ? 0 : ((current - baseline) / baseline * 100).toFixed(2);
+};
+
+// Test for proper rendering of all Anomaly ID Block data
+test('Anomaly ID block: all meta properties render', function(assert) {
+  let changeRate = calculateChangeRate(anomaly.current, anomaly.baseline);
+
+  this.set('anomaly', anomaly);
+  this.set('anomalyChangeRate', changeRate);
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: fyi, you could also use `this.setProperties({})` to set multiple properties

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  this.render(hbs`{{anomaly-id anomaly=anomaly}}`);
+
+  assert.equal(this.$(idSelector).text().trim(), anomalyId);
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: nit: can we add messages to these assertions. It'll make our life easier if one does fail. 
http://api.qunitjs.com/assert/equal

File: thirdeye/thirdeye-frontend/app/pods/application/template.hbs [hunk 5]
```diff
@@ -1 +1,6 @@
+<header class="nav-bar">
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/application/template.hbs]: Yes - some class names may change in the navbar PR.

File: thirdeye/thirdeye-frontend/app/pods/components/anomaly-id/template.hbs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<li class="anomaly-id__item anomaly-id__item--title">
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/components/anomaly-id/template.hbs]: Not using --metric, but yes using --value. The scss needed a bit of re-arranging.

File: thirdeye/thirdeye-frontend/app/styles/components/anomaly-id.scss (hunk 1 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/styles/components/anomaly-id.scss]: Sure. Lets do that.

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js (hunk 2 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: Not really, probably overkill.

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js (hunk 3 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: Thanks - I'll use that.

File: thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js (hunk 4 above)
Comment (newsummit) [inline thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js]: Agreed. Thanks.

**Pull request merged**